### ISSUE TITLE
Unable to build project after update to v2

### ISSUE THREAD

**Issue by burtek** [yesmeck/redux-modal#45]
After upgrading to build 2.0.0, I'm getting error:

```'redux-modal' does not contain an export named 'connectModal'```

There is no `index.d.ts` in package's root dir

**Comment by burtek** [comment]
Easiest solution is to add:

```
{
    ...
    "main": "./lib/index.js",
    "types": "./lib/index.d.ts"
}
```

 to `package.json` as per https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

**Comment by yesmeck** [comment]
Released 2.0.1

**Comment by burtek** [comment]
Thanks 👍
